### DISCUSSION TITLE
Jumping in Status buffer - possible?

### DISCUSSION THREAD

**Discussion by skrobul** [NeogitOrg/neogit#1604] [Q&A] [answered]
This question is mostly about keymaps in the Status popup/buffer. Is there any way to quickly jump to particular sections like `Unstaged changes`, `Staged changes` and so on?

For example, when opening status buffer like this:

![image](https://github.com/user-attachments/assets/e641c5c3-cbd8-4261-a2c2-2439c54d3489)

I would like to be able to go to a Staged changes or even better to a first file with just a single keystroke. I haven't used vim-fugitive for years now, but believe that's how it works there - you press `Ctrl-n` and cursor jumps to first unstaged file (`hosts/nucnix/configuration.nix`), then on another press it moves to first staged file (`hosts/nucnix/secrets/nullmailer_remotes_file.age`) and so on.

Is that possible with Neogit?

**Comment by CKolkey** [comment, marked as answer]
Oh, right on, good idea. Implemented here: da309da42fadb1d6a655f272cacf9dbb6a74558b

**Reply by skrobul** [reply]
Wow, that was quick, thank you! ❤️ 

fwiw, it works for my use case, but there is a minor bug - let me know if I should report it as bug. The jumps don't work if the cursor is placed on the line with hints,  `Head`, `Merge` or a line below that. Not a big deal, but if there is some easy mapping to be added it would be great :) Once again thanks!
